Core/Rewriters/Formatter.cs

```diff
@@ -74,6 +74,52 @@ public static T MarkWithFormatAnnotation<T> (T node)
       return baseCallNode.WithExpression (FormatInvocationExpression ((InvocationExpressionSyntax) node.Expression));
     }
 
+    public override SyntaxNode? VisitMethodDeclaration (MethodDeclarationSyntax node)
```

Comment:
I'd love to see "unit tests" for this method, might require more changes after you've added them though 😢.